index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header class="wrapper">
+        <nav class="navbar">
+           <p class="nav-left">Joe Doe</p>
+           <div class="hamberg">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+           </div>
+
+        </nav>
+
+        <div class="header-section">
+            <h1>Laura Jong</h1>
+            <h2>Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience!
+                 If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+        </div>
+
+        <div class="eclipse"></div>
```

Comment:
- [ ] Please make sure that any images that are supposed to be background images or a part of the aesthetics of the website, are supposed to **_be_** in the background and not present in the DOM itself.
- In other words, background images are supposed to be in the background and you can adjust the positioning by using [background styling properties](https://www.w3schools.com/css/css_background.asp).
- Kindly remove this div and add the background image to this section's container and adjust it to match the template.
- [HINT] You can use the [background image property](https://www.w3schools.com/cssref/pr_background-image.asp) to add an image as a background to a container.